src/cascadia/TerminalConnection/AzureConnection.cpp

```diff
@@ -25,6 +25,12 @@ using namespace winrt::Windows::Security::Credentials;
 
 namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
 {
+    // This file only builds for non-ARM64 so we don't need to check that here
+    bool AzureConnection::IsAzureConnectionAvailable()

```

Comment:
nit: Might want to add a note that this check is because the clientId is added by the release pipeline and might not be available in local builds

Reply:
Done